roles/confluent.variables/defaults/main.yml

```diff
@@ -558,10 +584,10 @@ ksql_monitoring_interceptors_enabled: "{{ monitoring_interceptors_enabled }}"
 
 # Control Center Variables
 
-### Only use to customize Linux User Control Center Service runs with. User must exist on host.
+### Only use to customize Linux User Control Center Service runs with. Default user is cp-control-center
```

Comment:
Modify this variable to customize the Linux User that the Control Center Service runs with. Default user is cp-control-center.